src/cmd/viper.go

```diff
@@ -117,7 +117,7 @@ func initViper() {
 	if err != nil {
 		// Config file not found; ignore
 		if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
-			message.Error(err, lang.CmdViperErrLoadingConfigFile)
+			message.Error(err, lang.CmdViperErrLoadingConfigFile, err)
```

Comment:
```suggestion
			message.Errorf(err, lang.CmdViperErrLoadingConfigFile, err)
```
message.Errorf would be even better/simpler to us